galpy/potential/amuse.py

```diff
@@ -158,13 +166,14 @@ def mass_density(self,x,y,z):
         HISTORY:
            2019-08-12 - Written - Webb (UofT)
         """
-        R= numpy.sqrt(x.value_in(units.kpc)**2.+y.value_in(units.kpc)**2.)
-        zed= z.value_in(units.kpc)
-        phi= numpy.arctan2(y.value_in(units.kpc),x.value_in(units.kpc))
+        R= numpy.sqrt(x.value_in(u.kpc)**2.+y.value_in(u.kpc)**2.)
+        zed= z.value_in(u.kpc)
+        phi= numpy.arctan2(y.value_in(u.kpc),x.value_in(u.kpc))
         return potential.evaluateDensities(self.pot,R/self.ro,zed/self.ro,
                                            phi=phi,t=self.tgalpy,
-                                           ro=self.ro,vo=self.vo) \
-                                           | units.MSun/(units.parsec**3.)
+                                           ro=self.ro,vo=self.vo,
+                                           use_physical=False
+                                           ) | u.MSun/(u.parsec**3.)
```

Comment:
This also needs a ``bovy_conversion.dens_in_msolpc3(self.vo,self.ro)``

Reply:
That also works.

Reply:
I've implemented this in a new commit.